meta-balena-common/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -0,0 +1,9 @@
+# We don't want grub modules in our sysroot, remove the entire prefix after do_deploy
+# This removes them for x86-64
```

Comment:
Don't we want this for 32 bits also?

Reply:
@floion IA32 EFI is a bit of a special case, and one that we're not really currently supporting. There is an open issue for this here: https://github.com/balena-os/meta-balena/issues/2097

The gist of it is that no new devices are being manufactured with IA32 EFI, though some very common devices circa ~2014 with Intel chips (Bay Trail, Cherry Trail, etc.) were only capable of booting with IA32 EFI even though the chips themselves were 64-bit. I've done a bit of work on supporting this, but it's a bit complicated. We would want to build `grub-efi` with the `i386-efi` target on x86_64. Consequently, it would likely be a separate recipe, like `lib32-grub-efi`, and the rest would be a standard x86_64 system.